cloudapp/src/app/print-slip-report/print-slip-report.service.ts

```diff
@@ -105,15 +54,15 @@ export class PrintSlipReportService {
 
 
   constructor(
-    private location: LocationService,
-    private restService: CloudAppRestService,
-    private router: Router,
+    location: LocationService,
+    router: Router,
```

Comment:
Why did you stop these from being a property?

Reply:
Both parameters are only used in the constructor, so there is no need to hold them as properties. But I can revert this, if this does not fit the code conventions. Just let me know.